CONTRIBUTING.md

```diff
@@ -1,6 +1,38 @@
 # Contributing
 First of all, thank you for contributing. It’s appreciated. This guide details how to use issues and pull requests to improve this project.
 
+Table of Contents
+=================
+
+  * [Requirements:](#requirements)
+  * [Installation](#installation)
+  * [Running](#running)
+  * [Making changes](#making-changes)
+    * [Branching](#branching)
+    * [Commit message](#commit-message)
+  * [Creating an Issue](#creating-an-issue)
+  * [Tests](#tests)
+
+## Requirements:
+* NodeJS(v4 LTS)
+* git crypt
+
+## Installation
+Clone this repository
+
+```bash
+$ git clone https://github.com/sphereio/sphere-product-type-import.git
+```
+
+We are using git-crypt for encrypting configs.
+Run
+
+```bash
+$ git-crypt unlock
+```
+
+to unlock the config file.
```

Comment:
@hisabimbola @emmenko @Siilwyn should we already address the git-crypt stuff we talked about on Friday in this PR? If I remember correctly we agreed on:
- removing git-crypt and credentials from the repository
- use environment variables in Travis for (integration) testing on CI
- developers should use their own project locally for testing

I am not sure if we were sure whether to use env vars or a credentials file locally, though 🤔